modules/client-configuration/pages/clients-openeuler.adoc

```diff
@@ -0,0 +1,107 @@
+[[clients-openeuler]]
+= Registering {openeuler} Clients
+
+This section contains information about registering traditional and Salt clients running {openeuler} operating systems.
+
+////
+ifeval::[{suma-content} == true]
+// Use list format: repository URLs / packages and metadata / version info
+[IMPORTANT]
+====
+* {openeuler} repository URLs are available from {scc}
+* Packages and metadata are provided by XXX, not by {suse}
+* For supported products, see the support table and the release notes
+====
+endif::[]
+////
```

Comment:
```suggestion
////

```
I commited the previous suggestion by accident.  It is still commented, though.  If possible, please fill in the actual info.  Thanks for your help!

Reply:
For reference, see https://github.com/uyuni-project/uyuni-docs/pull/1656 (which is about almalinux).